moon/rospy/rospy_rover.py

```diff
@@ -87,6 +87,8 @@ def register_handlers(self):
         self.steering_bl_publisher = rospy.Publisher('/' + self.robot_name + '/bl_steering_arm_controller/command', Float64, queue_size=QSIZE)
         self.steering_br_publisher = rospy.Publisher('/' + self.robot_name + '/br_steering_arm_controller/command', Float64, queue_size=QSIZE)
 
+        self.light_up_pub = rospy.Publisher('/' + self.robot_name + '/sensor_controller/command', Float64, queue_size=QSIZE, latch=True)
```

Comment:
is this really "light up"?? Sounds like some copy & paste error??

Reply:
I am not sure I understand the nature of the comment, does it express a concern? Should the name follow a different format? It has been chosen relatively arbitrarily.